File: CursoPOOUber/Php/Driver.php
<?php
require_once('Account.php');

class Driver extends Account{
    public function __construct($name, $document, $email, $password)
    {
        parent::__construct($name, $document, $email, $password);
    }

    public function printDataDriver(){
        echo "\nNombre del usuario: ".$this->name." Documento:".$this->document." Email: ".$this->email." password: ".$this->password; 
    }
}

?>

File: CursoPOOUber/Java/Driver.java
package Repositorio_POO.CursoPOOUber.Java;
class Driver extends Account{

    public Driver(String name, String document){
        super(name, document);
    }
    
    void printDataDriver(){
        System.out.println("Nombre de usuario: "+ name+
        "Documento: "+document);
    }
}
